plugins/modules/aci_fabric_scheduler.py

```diff
@@ -32,44 +28,44 @@
     aliases: [ descr ]
   recurring:
     description:
-    - If you want to make the Scheduler a recurring it would be a "True" and for a
-      oneTime execution it would be "False". For a shell just exclude this option from
-      the task
+    - If you want to make the scheduler a recurring operation, it should be set C(True) and for a one-time execution it should be C(False).
+    - For a shell, just exclude this option from the task.
     type: bool
   windowname:
     description:
-       - This is the name for your what recurring or oneTime execution
+    - The name of the schedule window.
+    - This is mandatory for the child class object B(trig:AbsWinddowP)
     type: str
   concurCap:
     description:
-       - This is the amount of devices that can be executed on at a time
+    - The amount of devices that can be executed on at a time.
     type: int
   maxTime:
     description:
-       - This is the amount MAX amount of time a process can be executed
+    - The maximum amount of time a process can be executed.
     type: str
   date:
     description:
-       - This is the date and time that the scheduler will execute
+    - The date and time that the scheduler will execute.
     type: str
   hour:
     description:
-       - This set the hour of execution
+    - The number of hours of execution.
     type: int
   minute:
     description:
-       - This sets the minute of execution, used in conjunction with hour
+    - The number of minutes of execution, used in conjunction with hour.
     type: int
   day:
     description:
-       - This sets the day when execution will take place
+    - The number of days when execution will take place.
     type: str
-    default: "every-day"
-    choices: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday','Sunday', 'even-day', 'odd-day', 'every-day']
+    default: every-day
+    choices: [ Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday, even-day, odd-day, every-day ]
```

Comment:
Should we align this input to also allow lower case inputs and avoid - usage? Thus providing more choices.

Reply:
Then, should I make a mapping in the constant.py file or should I just enlarge the current list locally because this attribute is the only one using these options?

Reply:
I would choose the constant file, because even though we don't have one now next time if somebody has similar input it can be reused. But this is only if we want to add, that was more a question on if that is a good idea?